packages/react-storage/src/components/StorageBrowser/views/LocationDetailView/LocationDetailView.tsx

```diff
@@ -92,6 +80,8 @@ export function LocationDetailView({
     onSearch,
   } = useLocationDetailView({ onNavigate: onNavigateProp, onExit });
 
+  const title = currentLocation?.prefix;
```

Comment:
Can just bring into ln 109 directly

Also, could you please confirm this logic is correct? A bucket type location may not have a prefix - the previous implementation fell back to bucket in such case. Also seems like you may need to use the the `currentPath` as well now to build the full path - not just the prefix (which would be the same as you navigated deeper into a location)

Reply:
I see, thanks for pointing that out. I'll address this issue alongside other views